lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -2257,12 +2255,7 @@ export class UserAgentApplication {
         if (loginStartPage) {
             // Cache the state, nonce, and login request data
             this.cacheStorage.setItem(Constants.loginRequest, loginStartPage, this.inCookie);
-            this.cacheStorage.setItem(Constants.loginError, "");
-
             this.cacheStorage.setItem(Constants.stateLogin, serverAuthenticationRequest.state, this.inCookie);
-
-            this.cacheStorage.setItem(Constants.msalError, "");
-            this.cacheStorage.setItem(Constants.msalErrorDescription, "");
```

Comment:
Do we not want to clear these?

Reply:
We are not clearing these, from what I see we are just creating empty entries even in non-error cases. I am doing some more testing for library created errors and may add it as appropriate. But these lines should go either way.